notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,581 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #76.        plt.grid(True, color="gray")
</pre><p>I ran the plots and the only axes that got the grid was the last one. Could you update the code to have the grid in all of them?</p><p>In addition to that could you modify the grid with the code below?</p>

<pre class="ql-syntax" spellcheck="false">plt.grid(":", alpha=0.5) # or alpha=0.3
</pre><p>This will make the grid visible but it will be more subtle. I use this as default in my plots and gives me good results. <code>":"</code> is the line style as dotted and <code>alpha=0.5</code> is a transparency of 50%. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>It looks better, Thanks.</p><p>Done.</p>